static/app/views/replays/detail/breadcrumbs/useBreadcrumbFilters.tsx

```diff
@@ -0,0 +1,89 @@
+import {useCallback, useMemo} from 'react';
+import uniq from 'lodash/uniq';
+import upperFirst from 'lodash/upperFirst';
+
+import {decodeList, decodeScalar} from 'sentry/utils/queryString';
+import getFrameDetails from 'sentry/utils/replays/getFrameDetails';
+import useFiltersInLocationQuery from 'sentry/utils/replays/hooks/useFiltersInLocationQuery';
+import {getFrameOpOrCategory, ReplayFrame} from 'sentry/utils/replays/types';
+import {filterItems} from 'sentry/views/replays/detail/utils';
+
+export type FilterFields = {
+  f_b_search: string;
+  f_b_type: string[];
```

Comment:
I wanna confirm that these are unique right across all the tabs?
The convention (that I made up) is:
- "f" for filter
- "b" in this case for breadcrumb tab
- and then the field name

Reply:
yes you are right!